WcaOnRails/vendor/assets/javascripts/jquery.wca-autocomplete.js

```diff
@@ -73,6 +75,13 @@
             return $div[0].outerHTML;
           },
 
+          regulation: function(regulation) {
+            var $div = $('<div class="wca-autocomplete-regulation"><span class="id"></span>: <span class="content_html"></span></div>');
+            $div.find(".id").text(regulation.id);
+            $div.find(".content_html").text($("<div/>").html(regulation.content_html).text());
```

Comment:
Cool, could we pull this out into a utility method like `wca.stripTags` or `wca.stripHtmlTags`? I think we could dump this in `application.js`.